psychotropic/cogs/structuregame.py

```diff
@@ -279,7 +282,7 @@ def can_be_ended(self, interaction):
     
     def end(self):
         """End a running game. This will cancel all pending tasks."""
-        for task in self.tasks:
+        for task in list(self.tasks):
```

Comment:
What is this supposed to solve? `.cancel()` just asks asyncio for task cancellation, it does not destroy the task object so `self.tasks` is not modified upon iteration